src/Autofac/Builder/RegistrationBuilder{TLimit,TActivatorData,TRegistrationStyle}.cs

```diff
@@ -366,13 +366,28 @@ public RegistrationBuilder(Service defaultService, TActivatorData activatorData,
         /// <returns>A registration builder allowing further configuration of the component.</returns>
         public IRegistrationBuilder<TLimit, TActivatorData, TRegistrationStyle> PropertiesAutowired(PropertyWiringOptions wiringFlags)
         {
-            var allowCircularDependencies = (int)(wiringFlags & PropertyWiringOptions.AllowCircularDependencies) != 0;
             var preserveSetValues = (int)(wiringFlags & PropertyWiringOptions.PreserveSetValues) != 0;
 
+            PropertiesAutowired(new DefaultPropertySelector(preserveSetValues), wiringFlags);
+
+            return this;
+        }
+
+        /// <summary>
+        /// Configure the component so that any properties whose types are registered in the
+        /// container and follow specific criteria will be wired to instances of the appropriate service.
+        /// </summary>
+        /// <param name="propertySelector">Selector to determine which properties should be injected</param>
+        /// <param name="wiringFlags">Set wiring options such as circular dependency wiring support.</param>
+        /// <returns>A registration builder allowing further configuration of the component.</returns>
+        public IRegistrationBuilder<TLimit, TActivatorData, TRegistrationStyle> PropertiesAutowired(IPropertySelector propertySelector, PropertyWiringOptions wiringFlags)
```

Comment:
if `wiringFlags` has the flag `PropertyWiringOptions.PreserveSetValues`, it won't do anything here. Is there any issue with that? Should it be in the summary or remarks?